### PR TITLE
GEOPY-74

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-74 - Fork the latest Simpeg development based on simpeg>=0.18**

### CODE REVIEW

File: geoapps/inversion/driver.py [hunk 1]
```diff
@@ -294,60 +296,70 @@ def get_regularization(self):
...
-                indActive=self.models.active_cells,
+                active_cells=self.models.active_cells,
                 mapping=wires.p,  # pylint: disable=no-member
-                gradientType=self.params.gradient_type,
+                gradient_type=self.params.gradient_type,
                 alpha_s=self.params.alpha_s,
-                alpha_x=self.params.alpha_x,
-                alpha_y=self.params.alpha_y,
-                alpha_z=self.params.alpha_z,
+                length_scale_x=self.params.alpha_x,
```
Comment (benk-mira) [inline geoapps/inversion/driver.py:303]: maybe we should follow suit with the rename?  more descriptive

File: geoapps/inversion/electricals/direct_current/two_dimensions/constants.py [hunk 2]
```diff
@@ -422,15 +408,13 @@
...
-    "resolution": None,
-    "detrend_order": None,
-    "detrend_type": None,
     "tile_spatial": 1,
     "out_group": {"label": "Results group name", "value": "DirectCurrentInversion"},
 }
 
 default_ui_json = dict(base_default_ui_json, **default_ui_json)
-
+for key in ["resolution", "detrend_order", "detrend_type", "y_norm", "alpha_y"]:
```
Comment (benk-mira) [inline geoapps/inversion/electricals/direct_current/two_dimensions/constants.py:416]: this might not be needed since we cut down the default_ui_json in the inversion params class:
image

File: geoapps/inversion/electricals/params.py [hunk 3]
```diff
@@ -143,3 +143,11 @@ def mesh(self):
     @mesh.setter
     def mesh(self, val):
         self.setter_validator("mesh", val, fun=self._uuid_promoter)
+
+    @property
```
Comment (benk-mira) [inline geoapps/inversion/electricals/params.py:147]: why is this necessary?

**Review by benk-mira — COMMENTED** [review]
This is great!  Just a couple comments to look at

File: geoapps/inversion/driver.py (hunk 1 above)
Comment (domfournier) [reply geoapps/inversion/driver.py:303]: Yup, done.

File: geoapps/inversion/electricals/direct_current/two_dimensions/constants.py (hunk 2 above)
Comment (domfournier) [reply geoapps/inversion/electricals/direct_current/two_dimensions/constants.py:416]: Correct, gone.

**Review by benk-mira — APPROVED** [review]
Nice, thanks for the updates!
